src/Content/OEmbed.php

```diff
@@ -184,13 +184,13 @@ private static function fetchURL(string $embedurl): \Friendica\Object\OEmbed
 
 		$eventDispatcher = DI::eventDispatcher();
 
-		$oembed_data = ['url' => $embedurl];
+		$oembed_data = ['data' => $oembed];
```

Comment:
Not providing an `url` key will break this hook, see https://github.com/friendica/friendica/blob/82ea62ba7cba358174a54de6f7ef341e4137263a/src/Core/Hooks/HookEventBridge.php#L137-L146

```suggestion
		$oembed_data = ['url' => $embedurl];
```